mailbox/postgres/src/test/java/org/apache/james/mailbox/postgres/DeleteMessageListenerTest.java

```diff
@@ -0,0 +1,56 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.postgres;
+
+import static org.apache.james.mailbox.postgres.PostgresMailboxManagerProvider.BLOB_ID_FACTORY;
+
+import org.apache.james.backends.postgres.PostgresExtension;
+import org.apache.james.mailbox.postgres.mail.PostgresMailboxManager;
+import org.apache.james.mailbox.postgres.mail.dao.PostgresMailboxMessageDAO;
+import org.apache.james.mailbox.postgres.mail.dao.PostgresMessageDAO;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class DeleteMessageListenerTest extends DeleteMessageListenerContract {
+    @RegisterExtension
+    static PostgresExtension postgresExtension = PostgresExtension.withoutRowLevelSecurity(PostgresMailboxAggregateModule.MODULE);
```

Comment:
do we need this test class anymore? I think it is redundant

Reply:
as my git comment "- Remove old test cases from PostgresMailboxManagerTest"

Why remove? because it will be easier when creating a contract with 2 test rls and non-rls

Reply:
Dont see anything wrong with that. We initially had test cases without rls. It's been detected that there was issues when enabling rls, so adding tests with rls too. Not redundant?

Reply:
but 2 test classes are almost the same. I wonder why not removing non-rls test and keeping rls?

Reply:
- We already have the RLS test for another class, then now add one more for `DeleteMessageListenerTest`, I think it is absolute normal
- It is just the same contract, but difference implementation
- The test code is free, we don't need to hesitate to limit it
- You can revert DeleteMessageListener code, and check the contract pass with NonRLS, but NO pass with RLS